service/src/main/java/bio/terra/externalcreds/services/TokenProviderService.java

```diff
@@ -37,6 +38,7 @@ public TokenProviderService(
         providerTokenClientCache,
         oAuth2Service,
         linkedAccountService,
+        fenceAccountKeyService,
```

Comment:
Do we still need the `fenceAccountKeyService` here? It doesn't look like it gets used

Reply:
Its needed for the `super` constructor.

Reply:
Oh, but its never used there either!

Reply:
Oh! Oops those lines were collapsed